docs/extensibility/theme-support.md

```diff
@@ -91,3 +91,63 @@ add_theme_support( 'disable-custom-colors' );
 ```
 
 This flag will make sure users are only able to choose colors from the `editor-color-palette` the theme provided or from the editor default colors if the theme did not provide one.
+
+## Editor styles
+
+A theme can provide a stylesheet to the editor itself, to change colors, fonts, and any aspect of the editor.
+
+### Add the stylesheet
+
+First thing you need to do is to enqueue the new editor style. Like this:
+
+```
+/**
+ * Enqueue block editor style
+ */
+function mytheme_block_editor_styles() {
+	 wp_enqueue_style( 'mytheme-block-editor-styles', get_theme_file_uri( '/style-editor.css' ), false, '1.0', 'all' );
+}
+add_action( 'enqueue_block_editor_assets', 'mytheme_block_editor_styles' );
+```
+
+Now create a new stylesheet, `style-editor.css` and save it in your theme directory.
```

Comment:
"Now" is a strange word here, this might read better without:

<pre>Create a new stylesheet, `style-editor.css`, in your theme directory.</pre>